### ISSUE TITLE
Limit number of organizations on collective page

### ISSUE THREAD

**Issue by xdamman** [opencollective/opencollective#1533]
From @hzoo: 

> is there a way for us to hide backers that have given a minimum amount from the main OC page for the project? Not sure if I'd want an org to be on there just because they gave $2? even an individual contributor could of given more in my mind

To mitigate that I suggest we limit to 30 instead of 100 the number of organizations to show on the collective page. Then we should just show the "load more" button.

**Comment by xdamman** [comment]
Actually, it should be a multiple of 4, so the top 20?

This will also speed up loading time of the page since there is less images to load.

**Comment by piamancini** [comment]
I agree this is a way to solve this immediate problem, but there's a larger conversation here. Why should a company that gives $2 have more real estate than a person who gives $1000? Even with the load more, companies are far more visible than individual backers.

**Comment by xdamman** [comment]
Showing the top 20 sets the bar at $1,250 to show up on the Babel page.
Whereas we show the top 100 backers on the page.

**Comment by piamancini** [comment]
I mean that the companies are easier to see, the backers are small and hard to see who's giving how much.
